doc/examples/numpy_operations/plot_structuring_elements.py

```diff
@@ -0,0 +1,58 @@
+"""
+=============================
+Generate structuring elements
+=============================
+
+This example shows how to use functions in :py:module:`skimage.morphology`
+to generate structuring elements.
```

Comment:
I think this paragraph could be improved by explaining what is a structuring element and providing few examples of functions using them (like erosion, dilation, functions in rank filters)...

Reply:
Hello sciunto. Thank you for pointing it out : )
I felt a bit hard to explain what structuring element is (my English is bad..).
I made this as an example, which modified from the current example of `dilation` function:
```python
>>> import numpy as np
>>> from skimage.morphology import dilation, square, disk
>>> bright_pixel = np.array([[0, 0, 0, 0, 0],
...                          [0, 0, 0, 0, 0],
...                          [0, 0, 1, 0, 0],
...                          [0, 0, 0, 0, 0],
...                          [0, 0, 0, 0, 0]], dtype=np.uint8)
>>> dilation(bright_pixel, square(3))
array([[0, 0, 0, 0, 0],
       [0, 1, 1, 1, 0],
       [0, 1, 1, 1, 0],
       [0, 1, 1, 1, 0],
       [0, 0, 0, 0, 0]], dtype=uint8)
>>> dilation(bright_pixel, disk(2))
array([[0, 0, 1, 0, 0],
       [0, 1, 1, 1, 0],
       [1, 1, 1, 1, 1],
       [0, 1, 1, 1, 0],
       [0, 0, 1, 0, 0]], dtype=uint8)
```
Not sure if it should be put into the code.